README.md

```diff
@@ -500,7 +500,7 @@ to test **DynamicCallCounter**:
 ```bash
 export LLVM_DIR=<installation/dir/of/llvm/13>
 # Generate an LLVM file to analyze
-$LLVM_DIR/bin/clang -O1 -emit-llvm -c <source_dir>/inputs/input_for_cc.c -o input_for_cc.bc
+$LLVM_DIR/bin/clang -O0 -Xclang -disable-O0-optnone -emit-llvm -c <source_dir>/inputs/input_for_cc.c -o input_for_cc.bc
```

Comment:
The new RUN line that you added suggests that you can safely remove ` -O0 -Xclang -disable-O0-optnone` :)

Reply:
That's right. I just updated that and fixed ```opt``` with ```-enable-new-pm=0```.